keps/draft-20180426-security-conformance.md

```diff
@@ -0,0 +1,136 @@
+---
+kep-number: draft-20180426
+title: Security Conformance
+authors:
+  - "@easeway"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-testing
+  - sig-api-machinery
+  - sig-network
+  - sig-node
+  - sig-cluster-lifecycle
+reviewers:
+  - "@tallclair"
+  - "@ericchiang"
+  - "@liggitt"
+  - "@davidopp"
+approvers:
+  - "@tallclair"
+  - "@ericchiang"
+  - "@liggitt"
+editor: "@easeway"
+creation-date: 2018-04-26
+status: provisional
+---
+
+# Security Conformance
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [List of Criteria](#list-of-criteria)
+* [Graduation Criteria](#graduation-criteria)
+
+## Summary
+
+Security Conformance defines citeria to certify a Kubernetes cluster to be minimally secure.
+
+## Motivation
+
+Configuring a Kubernetes cluster is complicated,
+and it's very easy to mis-configure a cluster to be insecure.
+The current Kubernetes conformance test suite barely covers security aspects.
+
+### Goals
+
+- Define the initial list (to evolve later) of criteria for minimal security requirements
```

Comment:
is this tied 1:1 to the security profiles? i.e. is there a minimal security profile that would pass these tests? which one?

Reply:
No. Security profile is more implementation specific, as it defines non-portable implementation specific rules like RBAC objects.